zuul.d/layout.yaml

```diff
@@ -1,40 +0,0 @@
----
```

Comment:
What does this have to do with Python 2 removal?

Reply:
In fact the entire zuul.d/ needs to removed as is not used by zuul.

Reply:
Remove it separately then